central/convert/storagetov2/compliance_profile_service.go

```diff
@@ -1,6 +1,7 @@
 package storagetov2
 
 import (
+	profileDS "github.com/stackrox/rox/central/complianceoperator/v2/profiles/datastore"
```

Comment:
This is the challenge with having the DS do the summary. Why not have the datastore return `[]*storage.Profile` and then convert it here (or service layer)?